### PR TITLE
Lesson7

### PR DESCRIPTION
**Pull request by ValentynaPiddubna**
Homework done

### CODE REVIEW

**Comment by ValentynaPiddubna** [comment]
Alexey, please review

File: Lesson7.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        else:
+            print(f"The {show_name} doesn't exist in the phone book. Please try again!!!")
+
+    elif command == 'list': # список всіх імен в книзі #
+        for key in phone_book:
+           print(key, phone_book[key])
+
+    elif command == 'stats': # кількість записів
+        if len(phone_book) == 1:
+            str = 'record'
```
Comment (klymenok) [inline Lesson7.py:42]: str is very bad name for variable since this word is reserved by Python

**Review by klymenok — APPROVED** [review]
Looks good, but check my comment

File: Lesson7.py (hunk 1 above)
Comment (ValentynaPiddubna) [reply Lesson7.py:42]: Thanks, I'll take to attention!

**Pull request merged**